plan/exhaust_physical_plans.go

```diff
@@ -177,36 +177,36 @@ func (p *LogicalJoin) getHashJoin(prop *requiredProp, innerIdx int) *PhysicalHas
 	return hashJoin
 }
 
-// joinKeysMatchIndex checks if all keys match columns in index.
-// It returns a slice a[] what a[i] means keys[i] is related with indexCols[a[i]].
+// joinKeysMatchIndex checks whether the join key is in the index.
+// It returns a slice a[] what a[i] means keys[i] is related with indexCols[a[i]], -1 for no matching column.
+// It will return nil if there's no column in the index.
```

Comment:
What do you mean by "there's no column in the index"?

Reply:
It should be a mistake. `if there's no column matches the index`